binderhub/event-schemas/launch.json

```
@@ -0,0 +1,25 @@
+{
+    "$id": "binderhub.jupyter.org/launch",
+    "version": 1,
+    "title": "BinderHub Launch Events",
+    "description": "BinderHub emits this event whenever a new repo is launched",
+    "type": "object",
+    "properties": {
+        "provider": {
+            "enum": [
+                "GitHub",
+                "GitLab",
+                "Git"
```

```suggestion
                "Git",
                "Gist"
```
Do we also need the gist provider?

---

We don't, it seems to just override github provider without changing name.